css/main.css

```diff
@@ -83,3 +93,112 @@ body {
   text-decoration: none;
   cursor: pointer;
 }
+
+
+
+
+#title
+{
+  position: fixed;
+  font-size: 80px;
+  padding-top: 2%;
+  right: 0;
+  left: 0;
+  z-index: -1;
+  width: 100%;
+  margin: auto;
+  text-align: center;
+  font-family: 'Ubuntu', sans-serif;
+  color: #000000;
+}
+
+/*Register Screen design*/
+.searchInput{
+  outline: none;
+  position: fixed;
+  height: fit-content;
+  border-radius: 5px;
+  border-color: #ffffff;
+  border-style: solid;
+  border-bottom-style: solid;
+  border-bottom-color: rgb(202, 201, 201);
+  padding: 1%;
+  width: 40%;
+  font-size: 40px;
+  top: 25%;
+  left: 15%;
+  margin: auto;
+  display: block;
+}
+
+
+
+.searchInput:hover,
+.searchInput:active,
+.searchInput:focus{
+  outline: none;
+  border-bottom-color: #000000;
+}
+
+
+.searchButton
+{
```

Comment:
doesn't matter if you do 
```css
style {
  //
}
```
or
```css
style
{
  //
}
```
but please stick to one or the other for consistency